quasar-osgi-tests/osgi-cacheable-one/src/main/java/org/testing/osgi/cacheable/ExampleCallable.java

```diff
@@ -1,14 +1,21 @@
 package org.testing.osgi.cacheable;
 
-import co.paralleluniverse.fibers.Suspendable;
+import org.testing.osgi.annotation.Suspendable;
 import java.util.concurrent.Callable;
+import java.util.function.Function;
 
 @SuppressWarnings("unused")
-public class ExampleCallable implements Callable<String> {
+public class ExampleCallable implements Callable<String>, Function<String, String> {
```

Comment:
Why do we need to bring in `Function` in this case? Isn't the existing `call` method enough? (I understand we want it to be annotated with `org.testing.osgi.annotation.Suspendable`)

Reply:
> Why do we need to bring in `Function` in this case? Isn't the existing `call` method enough? (I understand we want it to be annotated with `org.testing.osgi.annotation.Suspendable`)

Because merely _having_ a `@Suspendable` method is not enough. We also need a function that _invokes_ a `@Suspendable` method, which effectively involves implementing a `try-catch` block.